docs/getting-started-guides/ubuntu/operational-considerations.md

```diff
@@ -116,45 +116,52 @@ juju switch default
 
 ### Running privileged containers
 
-By default juju-deployed clusters do not support running privileged containers. If you need them, you have to edit ```/etc/default/kube-apiserver``` on the master nodes, and ```/etc/default/kubelet``` on your worker nodes. 
+By default juju-deployed clusters do not support running privileged containers.
```

Comment:
Since you're making changes, can you add a comma after default, so
"By default`,` juju-deployed clusters..."